<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rotate</class>
 <widget class="QWidget" name="Rotate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="no_rotation">
     <property name="text">
      <string>no rotation</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="clockwise_rotation">
     <property name="text">
      <string>rotate clockwise</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QRadioButton" name="counter_clockwise_rotation">
     <property name="text">
      <string>rotate counterclockwise</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="clockwise180_rotation">
     <property name="text">
      <string>rotate 180Â° clockwise</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QRadioButton" name="angle_rotation">
     <property name="text">
      <string>rotate by angle</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="degreeBox">
     <property name="maximum">
      <number>360</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
